src/main/java/com/devotedmc/ExilePearl/holder/PearlHolder.java

```diff
@@ -35,4 +38,19 @@
 	 * @return true if the holder is a block
 	 */
 	public boolean isBlock();
+
+	/**
+	 * Gets whether the holder is in a loaded chunk
+	 * @return true if the holder is in a loaded chunk
+	 */
+	public default boolean inLoadedChunk() {
+		Location loc = getLocation();
+		World world = loc.getWorld();
+
+		if (world == null) {
+			return false;
+		}
+
+		return world.isChunkLoaded(loc.getBlockX()/16, loc.getBlockZ()/16);
```

Comment:
That's not how chunk coords work. -1 and 1 are in different chunks, but divided by 16 both would be 0.

[Here](https://github.com/CivClassic/CivModCore/blob/master/src/main/java/vg/civcraft/mc/civmodcore/locations/chunkmeta/block/BlockBasedChunkMeta.java#L347) is what you should be using instead.